app/src/Pages/Home/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  getPublicData,
+  publicDataRequest,
+  allOffices,
+  availableCampaigns,
+  selectedOffices,
+  selectedCampaigns,
+  selectedStartDate,
+  selectedEndDate,
+  filteredPublicData,
+  campaignsTable,
+  mapData,
+  aggregatedContributorTypes,
+  aggregatedDonationSize,
+  aggregatedContributionsByRegion,
+  donationSizeByDonationRange,
+  setSelectedOffices,
+  setSelectedCampaigns,
+  setSelectedStartDate,
+  setSelectedEndDate,
+  summaryData,
+} from '../../state/ducks/publicData';
+import { showModal } from '../../state/ducks/modal';
+import Home from './Home';
+
+class HomePage extends React.Component {
```

Comment:
Also, is there a reason you're using class rather than function syntax? You could use react hook syntax to accomplish what `componentDidMount` is doing. I think it would be "lighter weight" than creating a full class component in this case, but Michael is better informed about these things than I.